styles.css

```diff
@@ -140,11 +145,11 @@
     filter: invert(1);
 }
 
-.switch-input:checked+.switch-label {
+.switch-input:checked + .switch-label {
```

Comment:
Please also avoid css combinators and generally everything that makes styles harder to override